config.py

```diff
@@ -229,7 +229,7 @@ def create_transaction(db):
     AUTHENTICATION_TYPE = "Database"
 
     # Build logs
-    BUILDLOGS_REDIS = {"host": "localhost"}
+    BUILDLOGS_REDIS = {"host": "localhost", "single_connection_client": False}
```

Comment:
Not really needed since it defaults to using the connection pool.

Reply:
Ah true